charts/internal/machine-controller-manager/seed/templates/vpa.yaml

```diff
@@ -17,10 +17,16 @@ spec:
       minAllowed:
         cpu: {{ .Values.resources.mcmProviderAWS.requests.cpu }}
         memory: {{ .Values.resources.mcmProviderAWS.requests.memory }}
+      maxAllowed:
```

Comment:
Can you add these to the `values` file and pass them via templates ? (similar to the minAllowed)

Reply:
Done